values/gitea/gitea.gotmpl

```diff
@@ -117,6 +117,16 @@ extraRootCA: |
   {{- $v.letsencryptCA | nindent 2 }}
 {{- end }}
 
+extraVolumes:
+- name: cert
+  secret:
+    secretName: letsencrypt-staging
+
+extraVolumeMounts:
+- name: cert
+  readOnly: true
+  mountPath: "/usr/local/share/ca-certificates"
+
```

Comment:
@Morriz Again, not sure why this has gotten in there, but should this stay or be reverted?

Reply:
me neither...just remove it